l16/dev/config.yaml

```diff
@@ -0,0 +1,69 @@
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
+# Genesis
+# ---------------------------------------------------------------
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 1
```

Comment:
We should decide on some value. If we have 100 genesis validators, then let's use `100` here.

Reply:
> I left comments. We should think about how to customize this script. How to provide availability to provide for example own address: `miner.etherbase=0xcce25d1620bd5e33485b808b30d3c805ea28dbe`

I think the script could help us with command that's why i added that script.

Reply:
> We should decide on some value. If we have 100 genesis validators, then let's use `100` here.

Thanks. I am changing it.